README.md

```diff
@@ -121,6 +121,11 @@ scores[0.5] = 4
 print(scores) //[3, 2, 4, 5]!
 ```
 
+## Comments
+A few langauges use `//` for commends, some use `#`, and some use `<!--` and `-->` for some reason, DreamBerd uses a much more intuitive commenting syntax, to comment out code, simply strikethrough the code you want to comment out
+
+<s>`~~badCode()!~~`</s>
+
```

Comment:
I think you should move this before the first comment is used, and then you also need to change every line where such a comment is used. Maybe you want to also point out that it's _not_ a multi-line comment (for historical reasons, of course). If `//` is still valid, the compiler/interpreter should give warnings for each usage, and/or you should mention that `~~` is the idiomatic way of commenting.

Reply:
`//` wouldn't be valid since it's a weird way to comment things out that nobody would use anyways